src/textual/widgets/_option_list.py

```diff
@@ -789,6 +842,22 @@ def get_option(self, option_id: str) -> Option:
                 f"There is no option with an ID of '{option_id}'"
             ) from None
 
+    def get_option_index(self, option_id):
+        """
+        Get the index of the option with the given ID.
+
+        Args:
+            option_id: The ID of the option to get the index of.
+        Raises:
+            OptionDoesNotExist: If no option has the given ID.
+        """
```

Comment:
Very minor, but please tidy the docstring a bit so it looks like the others :)

```suggestion
        """Get the index of the option with the given ID.

        Args:
            option_id: The ID of the option to get the index of.

        Raises:
            OptionDoesNotExist: If no option has the given ID.
        """
```